mach_nix/data/nixpkgs.py

```diff
@@ -19,7 +19,7 @@ class NixpkgsIndex(UserDict):
     # mapping from pypi name to nix key
     _aliases = dict(
         torch='pytorch',
-        tensorboard='tensorflowtensorboard'
+        tensorboard='tensorflow-tensorboard_2'
```

Comment:
```suggestion
```
This line is not needed at all anymore. It seems to be a leftover from an older version. I just tested and the current mach-nix version successfully detects tensorboard 1 & 2 from nixpkgs.